Adyen/Validators/BalanceChecker.swift

```diff
@@ -37,8 +37,13 @@ public struct BalanceChecker {
         public let isBalanceEnough: Bool
 
         /// :nodoc:
-        /// The remaining amount in the balance after payment. it is negative in case the `isBalanceEnough` is false.
-        public let remainingAmount: Payment.Amount
+        /// The remaining amount in the balance after payment.
+        /// it is at minimum zero when the available amount is less than the amount to pay.
```

Comment:
`it is at minimum zero when the available amount is less than the amount to pay.` -> `Zero, if available amount is less than the amount to pay` ?

Reply:
its important to say that its minimum is zero, your version doesn't say that explicitly

Reply:
I doesn't have to be one sentence :D

Reply:
I think balance (in case of money) already implies amount. Maybe `remainingBalance` is a better name. 
```
/// The remaining balance after the payment. <- first sentence
/// It is either a zero, if the available balance is not enough to pay the amount
/// or a positive number if the available balance is enough to pay the amount

Reply:
👍